packages/core/src/App/Containers/RealAccountSignup/choose-currency.js

```diff
@@ -1,11 +1,12 @@
-const currenciesPaymentAgentAvailability = (legal_allowed_currencies, all_payment_agent_list) => {
+const currenciesPaymentAgentAvailability = (legal_allowed_currencies, all_payment_agent_list, account_list) => {
     const result = [];
     legal_allowed_currencies.forEach(currency => {
         result.push({
             ...currency,
             has_payment_agent: !!all_payment_agent_list?.paymentagent_list?.list.filter(
                 agent => agent.currencies === currency.value
             ).length,
+            is_disabled: account_list?.filter(accouunt => accouunt.title === currency.value).map(a => a.is_disabled)[0],
```

Comment:
```suggestion
            is_disabled: account_list?.some(account => account.title === currency.value && a => a.is_disabled),
```
Can we use `.some`? It will return a boolean.